test_autofit/database/paths/test_switch.py

```diff
@@ -0,0 +1,11 @@
+import autofit as af
+
+
+def test_dynesty(session):
```

Comment:
This shows how to switch to using a database.

You can make a session something like this:

```python 
engine = create_engine(f"sqlite:///{database_path}")
session = sessionmaker(bind=engine)()
db.Base.metadata.create_all(engine)
```

we'll need a convenience function eventually